### PR TITLE
ursrv: remove google map and add leaflet heatmap

### PR DESCRIPTION
**Pull request by keevBush**
I remove google map for leaflet and leafletheatmap.
I added the data locally to test directly on the HTML template file.

### CODE REVIEW

**Comment by calmh** [comment]
I like it! Can you clean out the old google maps stuff (should be no references to it left)?

Two things about the actual heatmap though. First I think it's better if it starts zoomed out to the whole world, something like this:

Screenshot 2020-03-25 at 09 51 08

Secondly, I think the actual heat levels need some tweaking. We have a few hotspots, but much of the usage is pretty much invisible because there are lots of "weight 1" points that are just a very very faint blue tone... It would be nice if it were a bit more visible even for small usage sites

File: cmd/ursrv/static/index.html [hunk 1]
```diff
@@ -211,47 +216,48 @@
...
+      };
+      
+      var baseLayer = L.tileLayer(
+        'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',{
```
Comment (uok) [inline cmd/ursrv/static/index.html]: https ?

**Comment by uok** [comment]
Thanks, this fixes https://github.com/syncthing/syncthing/issues/6150

File: cmd/ursrv/static/index.html (hunk 1 above)
Comment (keevBush) [reply cmd/ursrv/static/index.html]: For second proposition can I have a json file for my test ???

**Comment by keevBush** [comment]
Can I have a json file with actual data for simply test

**Comment by calmh** [comment]
Sure let me sort that out

**Comment by calmh** [comment]
https://data.syncthing.net/locations.json

**Comment by calmh** [comment]
You can use that in the page instead of baking it into the template if that's easier / cleaner.

**Comment by AudriusButkevicius** [comment]
Very nice!

**Pull request merged**